workflow/scripts/sbs_process/apply_ic_field.py

```diff
@@ -0,0 +1,15 @@
+from lib.shared.illumination_correction import apply_ic_field
+from skimage.io import imread, imsave
+
+# load aligned image data
```

Comment:
I thought this was based on the parameter that you have below?

Reply:
I understand the confusion here. The helper functions in `lib/preprocess` take string paths to files then `imread` these, and `lib/sbs_process` functions want the actual np.array data as input. So for the `sbs_process` scripts it is necessary to load the files in the `scripts/` files then pass to helper functions.

Maybe we should choose one methodology to stay consistent with? Do you have a preference here? I think loading the files in `scripts/` might make more sense (than in the helper function) because we also save the files in `scripts/` with `imsave`

Reply:
I like reading and writing in scripts. Makes it easier to debug incorrect sizes.

I was more concerned with the segmentation cycle--I think it was specified as 1 or 0 last time, but now this looks good. you are relying on that in the file call here.